html-forms/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Form</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="">
+    <div>
+      <label for="email">Email<br>
```

Comment:
Great start! Visually, 100%. However: 

When wrapping an `input` tag with a `label` it is no longer necessary to use `for` on the `label`. Also, we want to avoid using `br` tags. While that looks correct visually, we want to use `div` tags for line break. This is something Brett touched on but I can elaborate more later. 

Looks like both of these are done a few times throughout the file. Please update and resubmit :) 





```
<label>Email
<input type="email" name="email" size="15" maxlength="30">
      </label>
```

Reply:
If you need to figure out how to get spacing without using the `br` you can use a `div` and in that case you may not wrap your `input` with a `label`:

```
<label for="email-address">Email</label>
<div>
  <input name="email" type="email" id="email-address">
</div>
```